test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/SentimentPredictionTests.cs

```diff
@@ -162,7 +162,8 @@ private BinaryClassificationMetrics EvaluateBinary(IHostEnvironment env, IDataVi
             var evaluator = new BinaryClassifierMamlEvaluator(env, new BinaryClassifierMamlEvaluator.Arguments());
             var metricsDic = evaluator.Evaluate(dataEval);
 
-            return BinaryClassificationMetrics.FromMetrics(env, metricsDic["OverallMetrics"], metricsDic["ConfusionMatrix"])[0];
+            return Microsoft.ML.Legacy.Models.BinaryClassificationMetrics
```

Comment:
>Microsoft.ML.Legacy.Models [](start = 19, length = 26)

does this need to be Microsoft.ML.Data?

Reply:
Actually before my change it was `BinaryClassificationMetrics` from `Microsoft.ML.Legacy.Models` but I had to add full name of class to distinguish between new `BinaryClassificationMetrics` from `Micorosft.ML.Data` and the legacy one.